src/renderer/components/AircraftSection/index.tsx

```diff
@@ -100,7 +100,10 @@ const index: React.FC<TransferredProps> = (props: AircraftSectionProps) => {
             const manifest = getCurrentInstall(Directories.inCommunity(props.mod.targetDirectory));
             console.log('Currently installed', manifest);
 
-            const track = _.find(props.mod.tracks, { url: manifest.source });
+            let track = _.find(props.mod.tracks, { url: manifest.source });
+            if (track === undefined) {
+                track = _.find(props.mod.tracks, { alternativeUrls: [manifest.source] });
```

Comment:
I don't think this would work, you need to find using an `url` property once for every `alternativeUrl`

Reply:
It does work, tested it

Reply:
This confuses the hell out of me but is correct.

Reply:
how does it even work

Reply:
Find in `props.mod.tracks` where `alternativeUrls` contains `manifest.source`